AllReadyApp/Web-App/AllReady/Dockerfile.debug

```diff
@@ -0,0 +1,12 @@
+FROM microsoft/aspnetcore-build:2.0
```

Comment:
@mk0sojo This whole dockerfile seems to do nothing more than just install the vs debugger and then on startup just keep the process running. What purpose does it serve?